### PR TITLE
54 judge evaluation requests

### PR DESCRIPTION
**Pull request by falusi94**

### CODE REVIEW

File: app/controllers/judgement_controller.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  def edit
+    @evaluation = Evaluation.find(params[:evaluation_id])
+  end
+
+  def update
+    unless SystemAttribute.evaluation_season?
+      return redirect_back fallback_location: judgements_path, alert: t(:not_evaluation_season)
+    end
+    evaluation = Evaluation.find(params[:evaluation_id])
+    success = CreateJudgement.call(evaluation, judgement_params)
```
Comment (salierri) [inline app/controllers/judgement_controller.rb]: Is it necessary to store the return value in a variable? You could just use the function in the condition.

File: app/services/create_judgement.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
+class CreateJudgement
+
+  def self.call(evaluation, params)
```
Comment (salierri) [inline app/services/create_judgement.rb]: Service objects are generally non-static classes, you instantiate it with your parameters, and then call `call` without any params. The advantage of that style is that you don't have to pass 'evaluation' and 'params' to every function manually, they can access them as a class variable

File: app/services/create_judgement.rb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  def self.create_message(evaluation, params)
+    old_point_request_status = evaluation.point_request_status
+    new_point_request_status = params[:point_request_status]
+    if old_point_request_status == new_point_request_status
+      point_request_change = 'nem változott'
+    else
+      point_request_change = "#{old_point_request_status} -> #{new_point_request_status}"
+    end
+
+    old_entry_request_status = evaluation.entry_request_status
```
Comment (salierri) [inline app/services/create_judgement.rb]: This part could be organized to a new function as it contains a considerable amount of duplicated code. Basically the only difference between the two is the `:point_request_status` key that could easily be a parameter.

File: app/controllers/judgement_controller.rb [hunk 4]
```diff
@@ -0,0 +1,33 @@
+class JudgementController < ApplicationController
```
Comment (salierri) [inline app/controllers/judgement_controller.rb]: If a controller name refers to a resource (even if it's not a model by itself) we should use plural forms.

**Comment by falusi94** [comment]
[RVT elnök tudjon elfogadni/elutasítani pontozás és belépőigényt](https://trello.com/c/teHQ2t1y/233-rvt-eln%C3%B6k-tudjon-elfogadni-elutas%C3%ADtani-pontoz%C3%A1s-%C3%A9s-bel%C3%A9p%C5%91ig%C3%A9nyt)

**Pull request merged**